src/node.cc

```diff
@@ -4385,34 +4386,27 @@ void Init(int* argc,
 
 
 struct AtExitCallback {
-  AtExitCallback* next_;
   void (*cb_)(void* arg);
   void* arg_;
 };
 
-static AtExitCallback* at_exit_functions_;
+static std::list<AtExitCallback*> at_exit_functions_;
```

Comment:
Use `AtExitCallback` not a pointer-to-`AtExitCallback`.  It's leaking memory now in `RunAtExit()`.

(Tiny nit: it should really be called `at_exit_functions`, it's not a class member.)